docker.go

```diff
@@ -154,10 +158,37 @@ func (p Plugin) Exec() error {
 		os.MkdirAll(dockerHome, 0600)
 
 		path := filepath.Join(dockerHome, "config.json")
-		err := os.WriteFile(path, []byte(p.Login.Config), 0600)
+		file, err := os.OpenFile(path, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0600)
```

Comment:
Can you help me understand why we need this change? If it is needed, the following error message is not accurate, since it says "Error writing" where this is just a `OpenFile` operation.

Reply:
We need this change to be able to append to the config file in case it already exists. I kept the error message generalized but I can fix it to be more specific.

Reply:
Thanks, the following error is probably sufficient in this case